### ISSUE TITLE
Quickstart Issues

### ISSUE THREAD

**Issue by grant** [GoogleCloudPlatform/functions-framework-php#8]
I ran into a couple issues running the quickstart for this package:

- Publish a non-dev Packagist version or require `"minimum-stability": "dev"` in `composer.json`
  - This is required for successfully installing the package.
- Unable to run the Functions Framework due to a bad arg:

```
Fatal error: Uncaught TypeError: Argument 1 passed to Google\CloudFunctions\HttpFunctionWrapper::__construct()
must be callable, string given, called in
/MY_DIR/vendor/google/cloud-functions-framework/src/Invoker.php on line 31
and defined in /MY_DIR/vendor/google/cloud-functions-framework/src/HttpFunctionWrapper.php:25

Stack trace: #0 /MY_DIR/vendor/google/cloud-functions-framework/src/Invoker.php(31): 
Google\CloudFunctions\HttpFunctionWrapper->__construct('helloHttp')
#1 /MY_DIR/vendor/google/cloud-functions-framework/router.php(60):
Google\CloudFunctions\Invoker->__construct('helloHttp', 'http')
#2 /MY_DIR/vendor/google/cloud-functions-framework/router.php(62): {closure}()
#3 {main} thrown in /MY_DIR/vendor/google/cloud-functions-framework/src/HttpFunctionWrapper.php on line 25
```

**Comment by bshaffer** [comment]
This is because whichever value you used in `FUNCTION_TARGET` was not a valid PHP function

**Comment by bshaffer** [comment]
Specifically, `helloHttp` has not been defined in your application. Is it possible your function is called `helloWorld`?

**Comment by grant** [comment]
I'm still not able to make the quickstart work as instructed.

My `index.php` looks like this:

```php
function helloHttp() {
    return "Hello World from PHP HTTP function!" . PHP_EOL;
};
```

This is from the README.

Are you able to follow the quickstart instructions in a new directory?

**Comment by bshaffer** [comment]
I believe the issue here was missing the `<?php` open tags. If that's the case, we can close this issue.

**Comment by bshaffer** [comment]
See `examples/hello/composer.json`:

```
{
    "require": {
        "google/cloud-functions-framework": "dev-master"
    }
}
```

This will work until we tag version `0.1`

**Comment by bshaffer** [comment]
Fixed in #12
